content/articles/wordpress-penetration-testing/index.md

```diff
@@ -0,0 +1,131 @@
+
+`wpscan --url http://example.com`
+
+![WPScan](/engineering-education/wordpress-penetration-testing/wpscan.jpg)
+
+Replace `example.com` with the URL of your WordPress site.
+
+#### Sqlmap
+Sqlmap is an extremely efficient penetration testing tool. It focuses on detecting SQL injection (SQLi) vulnerabilities on WordPress websites. Also, the Sqlmap tool helps developers to find SQL bugs present in themes and plugins. To use this tool, open the Kali Linux terminal and type this command:
+
+`sqlmap -u  www.example.com/index.html?parameter=1" --dbs`
+
+![Sqlmap](/engineering-education/wordpress-penetration-testing/sqlmap.jpg)
+
+The term `-dbs` will help you identify any SQLi bug found in the database during WP penetration testing.
+
+Therefore if there is no bug found, that doesn't signify that the site is safe. Sqlmap tool has multiple options to find SQLi vulnerabilities. To learn more, go through the Sqlmap documentation.
+
+#### PHPStan
+[PHPStan](https://phpstan.org/user-guide/getting-started) is an analysis tool that helps you discover bugs in your WordPress site. This tool comes as an extension designed for WordPress and which is downloaded separately. After configuring this tool, open the Kali Linux terminal and type this command:
+
+`vendor/bin/phpstan analyze Dir1 Dir2`
+
+![PHPStan](/engineering-education/wordpress-penetration-testing/phpstan.jpg)
+
+Then, replace `Dir1` and `Dir2` with the directories consisting of the WordPress code that you want to scan for bugs. To learn more, check out [PHPStan documentation](https://phpstan.org/user-guide/getting-started).
+
+#### XSSer
+[Cross-Site Scripting (XSS)]() is a common security threat found in WordPress websites. Hence, we can examine various WordPress themes and plugins for XSS bugs using a tool called Cross-Site "Scripter" (XSSer).
+
+XSSer is a framework to discover, exploit, and report XSS vulnerabilities on your WordPress websites. In addition, this tool enables you to bypass certain security filters and particular techniques of code injection.
+
+To do a simple scan, you can use a graphical interface. To do so, open the Kali Linux terminal and type this command:
+
+`xxser --gtk`
+
+![Xsser](/engineering-education/wordpress-penetration-testing/xxser.jpg)
+
+When the graphical interface opens, set the suitable options and enter the site's details, and WP pen-testing for XSS vulnerabilities will begin.
+
+### Conclusion
+I hope this guide has given an excellent introduction to WordPress penetration testing and practice as well. WordPress security is essential, and it's valuable to invest in the security of your WordPress website.
+
+Using the information provided in this document, you can come up with a WP pen-testing strategy. Now assemble your tools to perform reconnaissance and scanning, exploit where needed and mitigate vulnerabilities that you discover. Good luck!
```

Comment:
Create a " ### Further reading" section under  this  and provide three or links  for the  reader to learn more about WP pentesting.